app/views/admin/users/index.html.erb

```diff
@@ -1,40 +1,56 @@
+<% content_for :title_bar do %>
+  <%= link_to t('back'), edit_team_path(current_team), class: "btn btn-primary btn-sm navbar-btn" %>
+<% end %>
+
 <div class="row">
-  <div class="col-sm-12">
-    <h2><%= t('all users') %></h2>
+  <div class="col-xs-12 col-sm-10 col-sm-offset-1">
+    <div class="page-header">
+      <h4 class="page-header-title">
+        <i class="mi md-20">group</i> <%= t('all users') %>
+      </h4>
+    </div>
+  </div>
 
-    <table class="table table-hover">
-      <% @users.order(:name).each do |user| %>
-        <tr>
-          <td><%= user.name %></td>
-          <td><%= user.username %></td>
-          <td><%= user.initials %></td>
-          <td><%= user.role %></td>
-          <td><%= link_to user.email, "mailto:#{user.email}" %></td>
-          <td>
-            <div class="users_options btn-group pull-right">
-              <% if policy(user).update? %>
-                <%= link_to t('edit'), edit_admin_user_path(user),
-                  class: 'btn btn-default btn-sm' %>
-                <% if current_team.is_admin?(user) %>
-                  <%= link_to t('users.admin_off'), enrollment_admin_user_path(user, is_admin: false),
-                    data: {confirm: t('.are you sure you want to remove administration rights from this user')},
-                    method: :patch,
-                    class: 'btn btn-warning btn-sm' %>
-                <% else %>
-                  <%= link_to t('users.admin_on'), enrollment_admin_user_path(user, is_admin: true),
-                    data: {confirm: t('.are you sure you want to give administration rights to this user')},
-                    method: :patch,
-                    class: 'btn btn-primary btn-sm' %>
-                <% end %>
-                <%= link_to t('delete'), admin_user_path(user),
-                  data: {confirm: t('.are you sure you want to delete this user')},
-                  method: :delete,
-                  class: 'btn btn-danger btn-sm' %>
-              <% end %>
-            </div>
-          </td>
-        </tr>
-      <% end %>
-    </table>
+  <div class="col-xs-12 col-sm-10 col-sm-offset-1">
+    <div class="panel card">
+      <div class="panel-body">
+        <table class="table table-condensed table-striped table-hover admin-users-table">
+          <tbody>
+            <% @users.order(:name).each do |user| %>
+              <tr>
+                <td><%= user.name %></td>
+                <td><%= user.username %></td>
+                <td><%= user.initials %></td>
+                <td><%= user.role %></td>
+                <td><%= mail_to user.email %></td>
+                <td>
+                  <div class="users_options btn-group pull-right">
+                    <% if policy(user).update? %>
```

Comment:
Agreed! =) I am will be moving it to a presenter on a separate PR to refactor a couple views.